src/main/resources/jenkins/plugins/logstash/LogstashBuildWrapper/help-script.html

```diff
@@ -0,0 +1,35 @@
+<p>
+  With this script you can filter, modify and/or group messages that are sent to the
+  configured logstash backend. The following variables are available during execution:
+  <ul>
+    <li><strong>payload</strong> - JSONObject contaning the payload to be persisted.</li>
+    <li><strong>console</strong> - a class to output messages to build log without persisting them. It also works in a sandbox: <code>console.println("my logged message")</code>.</li>
+  </ul>
+</p>
+<p>
+  The script will be executed once per each line of build console output with the
+  variable <code>payload</code> set to a JSON payload with the "message" key being
+  an array with one string element - the current line.
```

Comment:
actually why a 1 element array?

Reply:
This is the format of the generated message JSON. At least when using the wrapper. User is free to add/modify lines. In the post-build action there are more elements.

Reply:
OK maybe we could clarify here that the whole payload as would be sent to the backend is available here. One could read the above as payload being a map with a single "message" key

Reply:
I tried to clarify the text. Let me know if it is better now. Or if you provide me with better wording, I would gladly update with it. I'm not a native english speaker.